mimic/rest/maas_api.py

```diff
@@ -642,14 +652,26 @@ def overview(self, request, tenant_id):
         """
         serves the overview api call,returns all entities,checks and alarms
         """
-        entities = self._entity_cache_for_tenant(tenant_id).entities_list
+        all_entities = self._entity_cache_for_tenant(tenant_id).entities_list
         checks = self._entity_cache_for_tenant(tenant_id).checks_list
         alarms = self._entity_cache_for_tenant(tenant_id).alarms_list
+        page_limit = min(int(request.args.get('limit', [100])[0]), 1000)
+        offset = 0
+        current_marker = request.args.get('marker', [None])[0]
+        if current_marker is not None:
+            index = _find_index_of_id(all_entities, current_marker)
+            offset = index if index != -1 else 0
+
+        entities = all_entities[offset:offset + page_limit]
+        next_marker = None
+        if offset + page_limit < len(all_entities):
+            next_marker = all_entities[offset + page_limit]['id']
+
         metadata = {}
         metadata['count'] = len(entities)
-        metadata['marker'] = None
-        metadata['next_marker'] = None
-        metadata['limit'] = 1000
+        metadata['marker'] = current_marker
```

Comment:
Just as a stylistic nitpick; this isn't a violation, but I prefer to do this:
```python
metadata = {
    'marker': current_marker,
    'next_marker': next_marker,
    # ...
}
```
since that puts the whole variable into an expression rather than splitting it into a bunch of statements and leaving the variable in a bunch of half-created states as it's being built up.